src/fixit/engine.py

```diff
@@ -103,14 +103,20 @@ def visit_hook(name: str) -> Iterator[None]:
         wrapper.visit_batched(rules)
         count = 0
         for rule in rules:
+            self.metrics[f"ViolationCount.{rule.name}"] = len(rule._violations)
+            self.metrics[f"ViolationCountWithReplacement.{rule.name}"] = 0
```

Comment:
What about just `Count` and `FixCount` for brevity? Esp in a potential world where auto-fixes aren't just strict "node replacements".